src/AuthorizationClient.js

```diff
@@ -799,10 +799,10 @@ class AuthorizationClient {
     return Utils.FormatAddress(ownerAddress);
   }
 
-  async Sign(message) {
+  async Sign(message, usePersonal=false) {
     return await Ethers.utils.joinSignature(
       this.client.signer.signDigest ?
-        await this.client.signer.signDigest(message) :
+        await this.client.signer.signDigest(message, usePersonal) :
```

Comment:
this must account for the case where the signer is not a remote signer, but is a local signer with a private key, which doesn't support this param.

I believe the default ethers.js signer will do a prefixed sign with `signer.signMessage`. you can determine if the signer is remote or not by either checking `signer.remoteSigner` to see if it's remote, or by checking `signer.signingKey` to see if it is not.

Reply:
or better yet, maybe we should change remote signer's signMessage method to be the method of signing with prefix, because that's what it's supposed to do.